src/components/Header.tsx

```diff
@@ -104,74 +98,89 @@ export const Header: React.FC = () => {
+          "navbar navbar-expand-sm navbar-dark navbar-primary",
+          styles.header
+        )}
+      >
+        <div className="container">
+          <Link href="/" className="navbar-brand">
+            <Image src={logo} height={60} alt="PrairieLearn logo" />
+          </Link>
+          <div className="d-flex flex-row">
+            <button
+              className="navbar-toggler"
+              type="button"
+              aria-controls="navbar"
+              aria-expanded={collapsed ? "false" : "true"}
+              aria-label="Toggle navigation"
+              onClick={() => setCollapsed((collapsed) => !collapsed)}
             >
-              <Dropdown.Item href="https://us.prairielearn.com/pl/login">
-                Main <span className="text-muted">(us.prairielearn.com)</span>
-              </Dropdown.Item>
-              <Dropdown.Item href="https://ca.prairielearn.com/pl/login">
-                Canada <span className="text-muted">(ca.prairielearn.com)</span>
-              </Dropdown.Item>
-            </DropdownButton>
-          </ul>
+              <span className="navbar-toggler-icon"></span>
+            </button>
+          </div>
+          <div
+            className={classnames("navbar-collapse", { collapse: collapsed })}
+            id="navbar"
+          >
+            <ul className="navbar-nav ms-auto mb-2 mb-sm-0">
+              <Dropdown as={NavItem}>
+                <Dropdown.Toggle
+                  as={NavLink}
+                  className={classnames(styles["nav-link"], {
+                    [`fw-bold ${styles.active}`]: useIsActive("/gallery"),
+                  })}
+                >
+                  Catalog
+                </Dropdown.Toggle>
+                <Dropdown.Menu>
+                  <NavDropdownItem href="/gallery/questions">
+                    Questions
+                  </NavDropdownItem>
+                  <NavDropdownItem href="/gallery/assessments">
+                    Assessments
+                  </NavDropdownItem>
+                  <NavDropdownItem href="/gallery/courses">
+                    Courses
+                  </NavDropdownItem>
+                  <NavDropdownItem href="/oer">OER</NavDropdownItem>
+                </Dropdown.Menu>
+              </Dropdown>
+              <Dropdown as={NavItem}>
+                <Dropdown.Toggle
+                  as={NavLink}
+                  className={classnames(styles["nav-link"], {
+                    [`fw-bold ${styles.active}`]: useIsActive("/about"),
+                  })}
+                >
+                  About
+                </Dropdown.Toggle>
+                <Dropdown.Menu>
+                  <NavDropdownItem href="/about">About Us</NavDropdownItem>
+                  <NavDropdownItem href="/research">Research</NavDropdownItem>
+                </Dropdown.Menu>
+              </Dropdown>
+              <li className="nav-item">
+                <RouterNavLink href="/contact">Support</RouterNavLink>
```

Comment:
I don't want to change this to "Support" until this actually references our support page. Alternatively, we can keep a "Contact" link that goes to `/contact` and add a "Support" link that goes to `/support`?

Reply:
I changed this back to Contact, and I will make this change in the next PR.